internal/plugins/ansible/init.go

```diff
@@ -0,0 +1,115 @@
+/*
+TODO(asmacdo) sdk licence header
```

Comment:
Could we use in the same standard used by helm?
- respect the kb license and keep them for all files that were created by copying past their ones
- add `Modifications copyright 2020 The Operator-SDK Authors` when we edit them
- keep our license when the file came not from kb.

Reply:
Yes, thats what these TODOs are for.